2_natural_language_processing/app/README.md

```diff
@@ -0,0 +1,40 @@
+# Full Stack Sentiment Analysis Application
+
+This app is a very basic implementation of an application that can take in reviews from the user, send those reviews to an LLM hosted on GCP VertexAI,
+and then make a prediction as to whether the sentiment of that review is positive or negative.
+
+There are two main components to this app:
+
+* the Python-based FastAPI backend service
+* the Svelte (JavaScript)-based frontend client (service)
+
+The frontend for the app looks like this:
```

Comment:
Please mention that this pattern is useful for aync use-cases where an immediate, low-latency result is not required. In fact, several requests can be batched up and sent to BigQuery.

We can later enhance this app so that the frontend makes a call to a low-latency model endpoint.